packages/components/src/tripInput/TripInput.js

```diff
@@ -38,7 +38,11 @@ export default function TripInput({
       <View style={[styles.container, style]}>
         {Platform.OS !== 'web' && <View style={styles.icon}>{inputIcon}</View>}
         <Text style={styles.label}>{label.length > 0 ? `${label}: ` : ''}</Text>
-        {value !== '' && <Text style={styles.value}>{value}</Text>}
+        {value !== '' && (
+          <Text numberOfLines={1} ellipsizeMode="clip" style={styles.value}>
```

Comment:
It doesn't seems to work on mobile. Text is not clipped when I try to set some smaller width to `TripInput`.

<img width="234" alt="Screenshot 2019-05-03 at 12 50 24" src="https://user-images.githubusercontent.com/2660330/57133099-1242ad80-6da2-11e9-90aa-8b7ab88de37f.png">

Reply:
@JoseDuSV That might be because the ellipsis is done on the internal text container..